src/envs/behavior.py

```diff
@@ -361,6 +371,7 @@ def set_igibson_behavior_env(self, seed: int) -> None:
             raise RuntimeError("ERROR: Failed to sample iGibson BEHAVIOR "
                                "environment that meets bddl initial "
                                "conditions!")
+        self.igibson_behavior_env.robots[0].initial_z_offset = 0.7
```

Comment:
oh - i see you just moved this line down here now. Does it need to be here? Can we just revert and keep it where it was?

Reply:
Yes it needs to be moved here. Since we are loading new scenes we need to set this after everytime we load a new scene not just on init.